### ISSUE TITLE
Force Google-Speech-Engine

### ISSUE THREAD

**Issue by desmeit** [vilic/cordova-plugin-tts#100]
The Samsung speech engine is pre-installed on my Samsung Test-Phone. I think this is the case with many users. The Google-Speech-Engine is much better. How can I force the Google-Speech-Engine to be installed automatically or give the user a link?

**Comment by ragcsalo** [comment]
Yeah, that would be nice :-)

**Comment by moghribalakrishna** [comment]
Looking forward to this, otherwise we can't keep the same voice across devices, very critical for usage

**Comment by qpwr** [comment]
I don't think this is possible. AFAIK the Google TTS engine doesn't come pre-installed on all devices, which is why the user would be required to install it before it can be used.

**Comment by desmeit** [comment]
You can force google voice if it is already installed:

`new TextToSpeech(cordova.getActivity().getApplication(), this, "com.google.android.tts");`
